app/grandchallenge/components/templates/components/componentinterface_list.html

```diff
@@ -59,27 +60,23 @@ <h2>{{ object_type|title }} {{ list_type|title }} Options</h2>
                     <td>{{ interface.description }}</td>
                     <td>{{ interface.get_kind_display }}</td>
                     <td><code>{{ interface.slug }}</code></td>
-                    {% if list_type == list_type_options.INPUT %}
-                        <td>
-                            {% if interface.is_image_kind %}
-                                <code>/input/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.mha</code>
-                                or
-                                <code>/input/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.tif</code>
-                            {% else %}
-                                <code>/input/{{ interface.relative_path }}</code>
-                            {% endif %}
-                        </td>
-                    {% elif list_type == list_type_options.OUTPUT %}
+
+                    {% with list_type|lower as url_segment %}
                         <td>
                             {% if interface.is_image_kind %}
-                                <code>/output/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.mha</code>
+                                <code>/{{ url_segment }}/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.mha</code>
                                 or
-                                <code>/output/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.tif</code>
+                                <code>/{{ url_segment }}/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.tif</code>
                             {% else %}
-                                <code>/output/{{ interface.relative_path }}</code>
+                                <code>/{{ url_segment }}/{{ interface.relative_path }}</code>
                             {% endif %}
                         </td>
-                    {% endif %}
+                    {% endwith %}
+                    <td>
+                        {% if interface.example_value %}
+                            {% include "components/partials/ci_example_value.html" %}
```

Comment:
Why make this a partial? It is not being reused anywhere.

Reply:
Done